src/v/config/tests/configuration_test.cc

```diff
@@ -0,0 +1,39 @@
+// Copyright 2025 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+#include "config/base_property.h"
+#include "config/configuration.h"
+
+#include <gtest/gtest.h>
+
+ss::logger lg("config test"); // NOLINT
```

Comment:
nit: probably doesn't matter much, but logger name without spaces is nicer (to specify in a command line etc.)